src/libraries/System.Resources.Extensions/tests/BinaryFormatTests/FormattedObject/BasicObjectTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Security.Cryptography;
+using BinaryFormatTests;
+
+namespace System.Resources.Extensions.Tests.FormattedObject;
+
+public class BasicObjectTests : Common.BasicObjectTests<FormattedObjectSerializer>
+{
+    private static readonly SHA256 s_md5 = SHA256.Create();
```

Comment:
At minimum...

```suggestion
    private static readonly SHA256 s_hash = SHA256.Create();
```

Then, be careful with `static` `HashAlgorithm` instances, as the type isn't thread-safe.  There's only one test here, so it doesn't really matter... (and I'm going to suggest getting rid of this anyways...)